content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,156 @@
+title: "How Does Timecode Vinyl Actually Work? (Pt. 1)"
+authors: Jan Holthuis
+tags: 2.3, timecode, dvs, vinyl control, serato, cd
+status: draft
+# Analog/Digital diagram via matplotlib: https://gist.github.com/Holzhaus/46ab8ea32d3b89025a0a640db171a14b
+
+
+Since the dawn of DJing, spinning vinyl records has never really gone out of fashion.
+Even when CDs became popular in 90s and most music listeners happily phased out their record players, the classical "two turntables and a mixer" setup stayed the epitome of DJing.
+In recent years, digital DJ controllers became more popular, but there are still many DJs that stick with their Technics SL-1210s turntables.
+
+There's a reason for that: the tactile feeling of manipulating music with your hands is something that controllers have had trouble archieving.
+Motorized controllers like the Rane One try to provide a similar workflow, but it's still to be seen if they will have a notable impact on the prevalence of vinyl.
```

Comment:
"prevalence of vinyl." Do you really wan to say that? For me this is orthogonal. A motorized controller is digital vinyl control without a fragile pick-up. It has potential to influence the controller marked. How about Mention Motorized controllers inside the DVS paragraph below? 

Here I suggest to reference the vinyl boom lately https://www.statista.com/chart/7699/lp-sales-in-the-united-states/
And maybe end into a rhetoric question of If this is driven by vinyl DJs? or such?

Reply:
> It has potential to influence the controller marked.

Yes, but it's still to be seen if they will replace vinyl setups or not. Right now, motorized controllers are pretty niche, even though the Numark NS7 has been in the market for a decade.

Reply:
Yes, If you tweak the sentence to this is makes sense.